sdk/src/main/scala/com/horizen/block/SidechainBlockHeader.scala

```diff
@@ -97,8 +95,7 @@ object SidechainBlockHeaderSerializer extends ScorexSerializer[SidechainBlockHea
     w.putInt(forgingStakeMerklePathBytes.length)
     w.putBytes(forgingStakeMerklePathBytes)
 
-    val vrfProofBytes = obj.vrfProof.bytes // TODO: replace with VRFProofSerializer... later
-    w.putBytes(vrfProofBytes)
+    VrfProofSerializer.getSerializer.serialize(obj.vrfProof, w)
```

Comment:
`ForgingStakeInfoSerializer` serialization logic above should be upgraded as well.
So, we should just pass the writer to it, and fix the logic inside the `ForgingStakeInfoSerializer`